apps/rush-lib/src/api/PackageJsonEditor.ts

```diff
@@ -7,7 +7,7 @@ import { Import, InternalError, IPackageJson, JsonFile, Sort } from '@rushstack/
 const lodash: typeof import('lodash') = Import.lazy('lodash', require);
 
 /**
- * @beta
+ * @public
  */
 export const enum DependencyType {
```

Comment:
Oh! I know it's not entirely relevant to this PR, but having that as `const` enum doesn't allow it to be used from JS scripts. I've been meaning to open an issue about that.

I.e `packageJsonEditor.addOrUpdateDependency(dep.name, dep.version, DependencyType.Peer);` fails at runtime because `DependencyType.Peer` is `undefined`

Reply:
Exporting const enums is bad practice in general. We should either replace this with a standard enum or a union of literals.

Reply:
Good catch -- fixed